package.json

```diff
@@ -25,26 +30,16 @@
     "tslib": "^1.10.0"
   },
   "devDependencies": {
+    "remark-cli": "^7.0.1",
+    "remark-preset-lint-recommended": "^3.0.3",
     "prettier": "^1.19.1",
     "typescript": "^3.7.5",
-    "rollup": "^1.18.0",
-    "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-sourcemaps": "^0.4.2",
-    "uglify-js": "^3.6.0"
+    "autorest": "^2.0.4413"
+  },
+  "remarkConfig": {
+    "plugins": [
+      "remark-preset-lint-recommended"
+    ]
   },
-  "files": [
```

Comment:
Why are we removing all here?

Reply:
It wasn't actually needed. We can put them back, but I'm mostly sure it was the same. Was there a difference?

Reply:
We should have them to avoid bundling trash

Reply:
Just 
`"files": [
    "lib/"
  ],` should be good.

Reply:
Done! @peloycosta @rtfpessoa